triton-vm/src/stark.rs

```diff
@@ -182,7 +186,8 @@ impl Stark {
         prof_start!(maybe_profiler, "ext tables");
         prof_start!(maybe_profiler, "LDE");
         master_ext_table.randomize_trace();
-        let fri_domain_ext_master_table = master_ext_table.to_fri_domain_table();
+        let (fri_domain_ext_master_table, ext_interpolation_polys) =
+            master_ext_table.to_fri_domain_table();
```

Comment:
I don't think we need to keep track of the polynomials if we have barycentric evaluation. File under: optimizable-if-bottleneck.

Reply:
How do we get the out-of-domain evaluation of the AET if we don't have the columns' interpolation polynomials (in coefficient form)?